lib/usd/pxrUsdPreviewSurface/overview.dox

```diff
@@ -7,7 +7,7 @@
 
 ## Overview
 
-This plugin contains a "pxrUsdPreviewSurface" Maya shader node that provides
+This plugin contains a "UsdPreviewSurface" Maya shader node that provides
```

Comment:
I feel like it's a bit unusual for a Maya node type to start with a capital letter? I don't have a strong opinion here, but I had the thought about maybe making the Maya node type name `usdPreviewSurface`, so I figured I'd mention it.